python_transport/setup.py

```diff
@@ -106,6 +106,7 @@ def get_systemd_lib(key):
             "wm-gw=wirepas_gateway.transport_service:main",
             "wm-dbus-print=wirepas_gateway.dbus_print_client:main",
             "wm-node-conf=wirepas_gateway.configure_node:main",
+            "wm-rtc=wirepas_gateway.rtc_service:main",
```

Comment:
Nope, it shouldn't be package with the transport wheel for now.
We should create a new wheel for BusClient and then different one for the other services